<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoList</class>
 <widget class="QWidget" name="ToDoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToDoList</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="3">
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5" rowspan="5">
       <widget class="QTableView" name="tv_done"/>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="btn_up">
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="cb_pending">
        <property name="currentText">
         <string/>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alpha</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QPushButton" name="btn_edit">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="doneLabel">
        <property name="text">
         <string>Done</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="5">
       <widget class="QTableView" name="tv_pending">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="6">
       <widget class="QPushButton" name="btn_del">
        <property name="text">
         <string>Del</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="pendingLabel">
        <property name="text">
         <string>Pending</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QPushButton" name="btn_done">
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="sortByEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>sort by</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="btn_down">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="11" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
